Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java

```diff
@@ -234,10 +237,10 @@ public CommandsAPI(String plugin) {
      * @param execute Callback for the command.
      */
     public void registerCommand(
-            String name,
-            String description,
-            List<ApplicationCommandOption> options,
-            Function1<? super Map<String, ?>, CommandResult> execute
+            @NonNull String name,
+            @NonNull String description,
```

Comment:
will these changes break all old plugin builds?

Reply:
No, these changes only add a marker for the IDE to indicate that values can't be null, so that we minimize runtime exceptions and improve development time.